web_responsive/static/src/js/web_responsive.js

```diff
@@ -529,12 +529,27 @@ odoo.define('web_responsive', function (require) {
 
     // DocumentViewer: Add support to maximize/minimize
     DocumentViewer.include({
-        events: _.extend(DocumentViewer.prototype.events, {
+        events: _.extend(_.omit(DocumentViewer.prototype.events, [
+            'keydown',
```

Comment:
I'd appreciate a comment explaining why are you modifying these events behavior.